RNTester/js/AsyncStorageExample.js

```diff
@@ -31,7 +31,7 @@ class BasicStorageExample extends React.Component<{}, $FlowFixMeState> {
   _loadInitialState = async () => {
     try {
       const value = await AsyncStorage.getItem(STORAGE_KEY);
-      if (value !== null) {
+      if (value) {
```

Comment:
We prefer the explicit null checking rather than truthy checking. For example, we don't want to be checking for the empty string since `"" == false`.

Reply:
This change was because of your review(`?String` intead of `String | null`).
As you know, `?String`(Maybe types) can also be null or undefined. so, `if (value !== null)` has error in Flow.

We should not use `Myabe types` to check for null. What would be better?

Reply:
@ifsnow sorry for taking a while to get back to you. Yeah, using `string | null` is fine. Do you mind changing it back to that? Then I can try to land this internally and see how many Flow errors it is causing there :D

Reply:
@cpojer I changed to use `string | null` instead of `?string` as you said. Please check. :)